src/sentry/tasks/post_process.py

```diff
@@ -1016,8 +1017,10 @@ def process_code_mappings(job: PostProcessJob) -> None:
         else:
             return
 
-        # XXX: We will stop calling data after we deploy this change
-        derive_code_mappings.delay(project.id, data=event.data, event_id=event.event_id)
+        if features.has("new-auto-source-code-config-queue"):
```

Comment:
I think you need to check your option here.

Reply:
Fixed.